ci-operator/step-registry/rhscl/deploy-and-test/rhscl-deploy-and-test-commands.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Config tempuser so Ansible runs smoothly
+echo "tempuser:x:$(id -u):$(id -g):tempuser:${HOME}:/bin/bash" >> /etc/passwd
+echo "tempuser:x:$(id -G | cut -d' ' -f 2):" >> /etc/group
+
+# Configure env for test run
+cp $KUBECONFIG /tmp/kubeconfig
+
+export KUBECONFIG=/tmp/kubeconfig
+export EXT_TEST=""
```

Comment:
is this being set in the tests?

Reply:
I can probably remove it since it's not getting set for rhscl, it holds the value for which tests need to be executed like different versions of dotnet or java since the same ansible-playbook is used to run those tests as well

Reply:
yeah if not used, you can remove and test it

Reply:
works fine after removing it